src/dirbrowserformaction.cpp

```diff
@@ -106,7 +106,9 @@ void DirBrowserFormAction::process_operation(Operation op,
 						status);
 					f->set("listpos", "0");
 					std::string fn = utils::getcwd();
-					fn.append(NEWSBEUTER_PATH_SEP);
+					if (fn.back() != '/') {
```

Comment:
I think it'd be better to turn `NEWSBEUTER_PATH_SEP` into a char (it's currently a string literal), and use it here. I don't think we'll ever turn that constant into an actual setting, and it's bound to disappear once we move to Rust, but in the meantime, it still feels wrong to hard-code the char here while still using the constant on the next line.

Reply:
Makes sense, done.

I have replaced most of the hardcoded uses of `'/'` except where it is used for something else than paths (e.g. urls or tags).